aws/ami/build_deb_ami.sh

```diff
@@ -18,23 +18,27 @@ source ../../SCYLLA-VERSION-GEN
 
 PRODUCT=$(cat build/SCYLLA-PRODUCT-FILE)
 DIR=$(dirname $(readlink -f $0))
+EXIT_STATUS=0
+DRY_RUN=false
 
 print_usage() {
     echo "build_deb_ami.sh --localdeb --repo [URL] --target [distribution]"
-    echo "  --localdeb  deploy locally built debs"
-    echo "  --repo  repository for both install and update, specify .repo/.list file URL"
+    echo "  --localdeb            deploy locally built debs"
+    echo "  --repo                repository for both install and update, specify .repo/.list file URL"
     echo "  --repo-for-install    repository for install, specify .repo/.list file URL"
     echo "  --repo-for-update     repository for update, specify .repo/.list file URL"
     echo "  --product             scylla or scylla-enterprise"
     echo "  --dry-run             validate template only (image is not built)"
     echo "  --download-no-server  download all deb needed excluding scylla from repo-for-install"
+    echo "  --log-file            Path for log. Default build/ami.log on current dir"
     exit 1
 }
 LOCALDEB=0
 DOWNLOAD_ONLY=0
-PACKER_SUB_CMD="build"
-
+PACKER_SUB_CMD="build -force -on-error=abort"
```

Comment:
why do you add "force" option?

Reply:
Hmmmm...
I did this also for GCE (also use packer).
Reading the docs https://www.packer.io/docs/commands/build
It looks good:

Forces a builder to run when artifacts from a previous build prevent a build from running. The exact behavior of a forced build is left to the builder. In general, a builder supporting the forced build will remove the artifacts from the previous build. This will allow the user to repeat a build without having to manually clean these artifacts beforehand.